### PR TITLE
Templates: Add Transifex link

### PR DESCRIPTION
**Pull request by tooomm**
## Short roundup of the initial problem
Not super easy to get involved.

## What will change with this Pull Request?
- Add link to our transifex project page which handles all strings and languages.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -3,3 +3,6 @@ contact_links:
   - name: 💬 Discord Community (Get help with server issues, e.g. Login)
     url: https://discord.gg/3Z9yzmA
     about: Need help with using the client? Want to find some games? Try the Discord server!
+  - name: 🌐 Translations (Help to improve the localization of the app)
+    url: https://www.transifex.com/cockatrice/cockatrice/
+    about: For more information and guidance check the "Translation FAQ" in our GitHub Wiki!
```
Comment (ebbit1q) [inline .github/ISSUE_TEMPLATE/config.yml:8]: can we link to it here?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -3,3 +3,6 @@ contact_links:
   - name: 💬 Discord Community (Get help with server issues, e.g. Login)
     url: https://discord.gg/3Z9yzmA
     about: Need help with using the client? Want to find some games? Try the Discord server!
+  - name: 🌐 Translations (Help to improve the localization of the app)
```
Comment (ebbit1q) [inline .github/ISSUE_TEMPLATE/config.yml:6]:
```suggestion
  - name: 🌐 Translations (Help improve the localization of the app)
```

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (tooomm) [reply .github/ISSUE_TEMPLATE/config.yml:8]: We can try, I'm not sure if it's supported though.

**Review by ebbit1q — APPROVED** [review]
looks good, lets test if that link works

**Comment by ebbit1q** [comment]
and it does not work

**Comment by ebbit1q** [comment]
try html tags instead of markdown?

**Comment by tooomm** [comment]
>try html tags instead of markdown?

I'm not around a computer for a while, give it a go!

**Pull request merged**